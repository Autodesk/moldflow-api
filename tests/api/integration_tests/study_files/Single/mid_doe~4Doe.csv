 ATTENTION - THIS DOCUMENT UNIT SYSTEM IS: SI
---------------------------
 Study: mid_doe.sdy
---------------------------
Analysis of Influences 

Experimental data
Variables:,Dimension scale factor,Melt temperature,Qualities:,Standard Deviation:Bulk temperature at end of fill,Clamp force,Injection pressure,Wall shear stress,Standard Deviation:Temperature at flow front,Standard Deviation:Time to reach ejection temperature
Runs:\Units:,%,K,,K(d),N,Pa,Pa,K(d),s
0,-5,560.15,,1.7628,16206.9,5.17047e+07,845119,1.21083,0.517601
1,-5,600.15,,1.20543,7182.07,2.9668e+07,581270,0.304797,0.576798
2,5,560.15,,1.68139,15964.2,5.12216e+07,861348,1.06475,0.562444
3,5,600.15,,1.20707,7785.98,2.98535e+07,582161,0.287863,0.567158

Percentage influences
Qualities:,Standard Deviation:Bulk temperature at end of fill,Clamp force,Injection pressure,Wall shear stress,Standard Deviation:Temperature at flow front,Standard Deviation:Time to reach ejection temperature
Dimension scale factor,7.17621,2.05664,0.681033,3.05629,8.83124,35.518
Melt temperature,92.8238,97.9434,99.319,96.9437,91.1688,64.482


---------------------------
