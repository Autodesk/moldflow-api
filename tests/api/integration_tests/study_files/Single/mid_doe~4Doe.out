1
0
0
2
1
(C)2009 2010 2011 2012 2013 2014 2015 2016 
0
2
1
(C)2017 2018 2019 2020 2021 2022 2023 2024 
0
2
1
(C)2025 
0
3
0
0
0
0
0
1600001
0
0
10
2
Autodesk Moldflow Insight 2026
49.1.198
0
14
0
0
19
3
ADSKPW0F99G8
Windows 11 ( Build 26100 )
GenuineIntel Intel64 Family 6 Model 170 Stepping 4 ~2995 MHz
3
16
22
32265.097656
9999988
1
1761929655
0
104600
1
1761929656
0
104601
1
1761929656
0
1650004
0
0
1650001
0
0
1650002
0
0
1650100
0
0
1650102
0
0
1650110
0
1
2
1650111
0
1
6
1650144
0
0
0
0
0
1650143
1
Dimension scale factor
0
1650147
3
%
%
%
2
-5
5
0
0
0
1650143
1
Melt temperature
0
1650147
3
K
K
K
2
560.15002441
600.15002441
1650148
0
0
0
0
0
1650143
1
Standard Deviation:Bulk temperature at end of fill
0
1650143
1
Clamp force
0
1650143
1
Injection pressure
0
1650143
1
Wall shear stress
0
1650143
1
Standard Deviation:Temperature at flow front
0
1650143
1
Standard Deviation:Time to reach ejection temperature
0
1650003
0
0
1650108
0
0
1650112
0
1
4
1650005
0
0
1650006
0
0
1650130
0
0
1650131
0
4
0
4
0
0
1650131
0
4
0
4
0
0
1650131
0
4
0
4
0
0
1650131
0
4
0
3
1
0
1650131
0
4
0
2
2
0
1650131
0
4
0
2
1
1
1650131
0
4
0
2
0
2
1650131
0
4
0
2
0
2
1650131
0
4
0
2
0
2
1650131
0
4
0
1
1
2
1650131
0
4
0
0
2
2
1650131
0
4
0
0
1
3
1650131
0
4
0
0
0
4
0
0
0
0
0
0
1650007
0
0
1650010
0
0
1650140
1
Standard Deviation:Bulk temperature at end of fill
1
1
1650165
0
0
1650141
1
Dimension scale factor
1
7.1762137413
1650141
1
Melt temperature
1
92.823783875
1650140
1
Clamp force
1
2
1650160
0
0
1650141
1
Dimension scale factor
1
2.0566372871
1650141
1
Melt temperature
1
97.943359375
1650140
1
Injection pressure
1
3
1650160
0
0
1650141
1
Dimension scale factor
1
0.68103259802
1650141
1
Melt temperature
1
99.318969727
1650140
1
Wall shear stress
1
4
1650160
0
0
1650141
1
Dimension scale factor
1
3.0562887192
1650141
1
Melt temperature
1
96.943710327
1650140
1
Standard Deviation:Temperature at flow front
1
5
1650165
0
0
1650141
1
Dimension scale factor
1
8.8312425613
1650141
1
Melt temperature
1
91.168754578
1650140
1
Standard Deviation:Time to reach ejection temperature
1
6
1650165
0
0
1650141
1
Dimension scale factor
1
35.518016815
1650141
1
Melt temperature
1
64.481987
1650020
0
0
1650021
0
0
1650013
0
0
9999989
2
1761929655
1761929746
1
9.828125
